packages/extension/src/manifest.json

```diff
@@ -10,7 +10,7 @@
     "48": "icons/icon-48.png",
     "128": "icons/icon-128.png"
   },
-  "incognito": "spanning",
+  "incognito": "split",
```

Comment:
Why are we changing it in this PR?

Reply:
One of the issues noticed while testing the feature was data was not being sent to the DevTools UI from the service worker in incognito mode. Thats why the change is there in the diff.